src/encoded/static/components/cart/cart.js

```diff
@@ -1571,6 +1571,8 @@ const processFilesAnalyses = (files, analyses) => {
  * Sort an array of files by their statuses according to their order in
  * pseudoDefaultFileStatusOrder. Anything not in pseudoDefaultFileStatusOrder gets sorted randomly
  * at the end.
+ * @param {array} files Files to sort
+ * @return {array} Files sorted by the three following statuses
  */
 const pseudoDefaultFileStatusOrder = ['released', 'archived', 'in progress'];
 const sortPseudoDefaultFilesByStatus = (files) => (
```

Comment:
Lines 1577 and 1578 should be swapped to make align sortPseudoDefaultFilesByStatus with the jsdoc comment

Reply:
I moved `pseudoDefaultFileStatusOrder` above all this to avoid doing a static assignment inside a function.